### PR TITLE
feat: turbo eth synthetic token

### PR DESCRIPTION
**Pull request by daniel-savu**
Reuses the eth logo until we get a custom tETH one

### CODE REVIEW

File: deployments/warp_routes/tETH/metadata.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Turbo ETH",
+  "symbol": "tETH",
+  "description": "Warp Route tETH",
+  "image": "",
```
Comment (paulbalaji) [inline deployments/warp_routes/tETH/metadata.json:5]: no image?

Comment (tkporter) [inline deployments/warp_routes/tETH/metadata.json:5]: does this need to point to the logo or no?

**Review by tkporter — APPROVED** [review]

File: deployments/warp_routes/tETH/metadata.json (hunk 1 above)
Comment (daniel-savu) [reply deployments/warp_routes/tETH/metadata.json:5]: couldn't find any logo, just asked them about it. The warp UI can be updated when one becomes available luckily

Comment (daniel-savu) [reply deployments/warp_routes/tETH/metadata.json:5]: this is a custom format I used for synthetic metadata based on an example somewhere, all fields are optional

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**